Sources/UAPusher/Builder/Campaigns.swift

```diff
@@ -0,0 +1,51 @@
+
+import JSON
+
+/// Campaigns segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/api/ua/#campaigns-object
+public final class Campaigns: BuildingBlock {
+    
+    // MARK: Class fields
+    
+    /// Campaign representation
+    var payload: JSON
+    
+    // MARK: Predefined values
+    
+    public enum Predefined: JSONRepresentable {
+        case test
```

Comment:
What's your thoughts behind this test case? I think it would be more clean to have `makeJSON` just return an empty `categories` array or loosen up the protocol a bit if the methods are not needed in every building block.